data/World/Spirit Temple MQ.json

```diff
@@ -88,7 +112,21 @@
                 is_adult",
             "Spirit Temple MQ GS Sun Block Room": "
                 (logic_spirit_mq_sun_block_gs and Boomerang and
-                (can_play(Song_of_Time) or logic_spirit_mq_sun_block_sot)) or
+                    (can_play(Song_of_Time) or logic_spirit_mq_sun_block_sot)) or
+                is_adult",
+            "Spirit Temple MQ Child Climb Pot": "(Small_Key_Spirit_Temple, 6)",
+            "Spirit Temple MQ Central Chamber Floor Pot 1": "True",
+            "Spirit Temple MQ Central Chamber Floor Pot 2": "True",
+            "Spirit Temple MQ Central Chamber Floor Pot 3": "True",
+            "Spirit Temple MQ Central Chamber Top Left Pot 1": "can_use(Hover_Boots) or can_play(Song_of_Time)",
+            "Spirit Temple MQ Central Chamber Top Left Pot 2": "can_use(Hover_Boots) or can_play(Song_of_Time)",
```

Comment:
Requested Change?: Whichever of these pots is on the right, it might be safe in default logic to obtain it with the Boomerang. The actual Boomerang throw is fairly easy, certainly Hookshotting the Skulltula in this location in vanilla is a bit harder. You have to have a jumpslash item (sword or sticks) to reach the hand without Song of Time as child, which is easy also, and some jumpslashes for more distance (see: vanilla adult Spirit boulder room) are already considered default. So these are two things that are slightly awkward but would probably be default logic on their own. Why don't you take a look and tell me what you think?

Trick: The left pot can also be obtained with a Boomerang trick throw, but it's actually pretty hard to do.

Reply:
Yeah I think it's easy enough w/ Boomerang.

Reply:
The trick jump to the upper ledges only works as adult.

I gave you a note in the previous message in this thread about how to implement the Spirit logic for Boomeranging the pot, but you didn't follow it.

Reply:
You still need to fix the trick jump to the left pot.